### PR TITLE
fix windows target

### PR DESCRIPTION
**Pull request by LProulx86**
GitHub Issue: #

## Proposed Changes

 - [x] Bug fix
 - [ ] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build or CI related changes
 - [ ] Documentation content changes
 - [ ] Other, please describe:

## What is the current behavior?

Last update didn't update windows target in winUI project. winUI target 18362

## What is the new behavior?

WinUI now targeting 19041

## Impact on version

- [ ] **Major** (Public API was modified.)
  - Public constructs (class, struct, delegate, enum, etc.) were removed or renamed.
  - Public members were removed or renamed.
  - Public method signatures were changed or renamed.
- [ ] **Minor** (Public API was extended.)
  - Public constructs, members, or overloads were added.
- [x] **Patch** (Public API was unchanged.)
  - A bug in behavior was fixed.
  - Documentation was changed.
- [ ] **None** (The library is unchanged.)
  - Only code under the `build` folder was changed.
  - Only code under the `.github` folder was changed.

## Checklist

Please check that your PR fulfills the following requirements:

- [x] Documentation has been added/updated.
  - README.md was reviewed.
  - [BREAKING_CHANGES.md](../BREAKING_CHANGES.md) was updated (if you introduced a breaking change).
- [ ] Automated Unit / Integration tests for the changes have been added/updated.
- [x] Your conventional commits are aligned with the **Impact on version** section.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
...
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+## 2023-12-05
+
+### Added
+
+### Changed
+net7.0-windows10.0.18362 to net7.0-windows10.0.19041
```
Comment (Guidemarcus) [inline CHANGELOG.md]: please make full sentences

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```
Comment (Guidemarcus) [inline CHANGELOG.md]: no need to duplicate all the sections. You can simply add up to what is already there

File: src/MessageDialog.Uno.WinUI/MessageDialog.Uno.WinUI.csproj [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
 	<PropertyGroup>
-		<TargetFrameworks>net7.0;net7.0-windows10.0.18362;net7.0-android;net7.0-ios;net7.0-maccatalyst;net7.0-macos</TargetFrameworks>
```
Comment (jeanplevesque) [inline src/MessageDialog.Uno.WinUI/MessageDialog.Uno.WinUI.csproj:3]: This change is another breaking change that should be combined with the previous one (change to .NET 7). It should be listed in the BREAKING_CHANGES.md file.

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

File: src/MessageDialog.Uno.WinUI/MessageDialog.Uno.WinUI.csproj (hunk 3 above)
Comment (LProulx86) [reply src/MessageDialog.Uno.WinUI/MessageDialog.Uno.WinUI.csproj:3]: This is already list in BREAKING_CHANGES. This file wasn't been update on last PR.

**Review by carlh98 — APPROVED** [review]

**Review by Guidemarcus — APPROVED** [review]

**Review by jeanplevesque — APPROVED** [review]

**Pull request merged**